source/server/server.cc

```diff
@@ -366,24 +366,24 @@ void InstanceImpl::run() {
   config_->clusterManager().shutdown();
   handler_.reset();
   thread_local_.shutdownThread();
-  ENVOY_LOG(warn, "exiting");
+  ENVOY_LOG(info, "exiting");
   ENVOY_FLUSH_LOG();
 }
 
 Runtime::Loader& InstanceImpl::runtime() { return *runtime_loader_; }
 
 void InstanceImpl::shutdown() {
-  ENVOY_LOG(warn, "shutdown invoked. sending SIGTERM to self");
+  ENVOY_LOG(info, "shutdown invoked. sending SIGTERM to self");
   kill(getpid(), SIGTERM);
 }
 
 void InstanceImpl::shutdownAdmin() {
-  ENVOY_LOG(warn, "shutting down admin due to child startup");
+  ENVOY_LOG(info, "shutting down admin due to child startup");
   stat_flush_timer_.reset();
   handler_->stopListeners();
   admin_->mutable_socket().close();
 
-  ENVOY_LOG(warn, "terminating parent process");
```

Comment:
Same for the log messages on the way out of the process.

Reply:
done